package.json

```diff
@@ -27,8 +27,8 @@
     "watch": "tsc --watch",
     "docs": "npx typedoc --options typedoc.json js/index.ts",
     "test": "jest",
-    "prepublishOnly": "./scripts/node_prepublish.py",
-    "postpublish": "./scripts/node_postpublish.py"
+    "prepublishOnly": "../../scripts/node_prepublish_autoevals.py",
```

Comment:
Moved these scripts to the internal repo, because they are no longer isolated to this one.